rules/S6678/metadata.json

```
@@ -0,0 +1,19 @@
+{
+  "title": "Use PascalCase for named placeholders in structured logging",
```

We don't detect if they use the structured logging functionality. So, I would not mention it.
```suggestion
  "title": "Use PascalCase for named placeholders",
```